### ISSUE TITLE
normalization methods when calculate the metrics

### ISSUE THREAD

**Issue by jvzihaha** [yixin26/SVR-SP#1]
May I know the normalization methods when you calculate the metrics reported in the papers ?
I tried unit cube and unit sphere, all of them can't get the reported metrics.
I checked the code in DISN,  tried sphere with radius 1, but still  can't get the reported values.
Appreciated if anyone could help.

**Comment by jvzihaha** [comment]
I mean the CD and EMD metrics

**Comment by yixin26** [comment]
Hi,

In preprocessing step, all shapes from the ShapeNetCore dataset are normalized to a unit sphere. Then point sampling and signed distance computation for training are performed in the normalized shape space.

In this way, the generated meshes already lie within a **canonical** frame and so as the ground truth meshes.
Thus normalization is no longer required for CD and EMD calculation.

Here is the source code for shape normalization in the preprocessing step:
[def get_normalize_mesh(model_file, norm_mesh_sub_dir)](https://github.com/laughtervv/DISN/blob/f8206adb45f8a0714eaefcae8433337cd562b82a/preprocessing/create_point_sdf_grid.py#L169)

To reproduce the results, I recommend you to use [this script](https://github.com/yixin26/SVR-SP/blob/main/code/sdf2obj.py) for generation and [this script](https://github.com/laughtervv/DISN/blob/master/test/test_cd_emd.py) for computing CD and EMD.

If you want to modify the code, then make sure to process the training set and the testing set in the same way.

**Comment by yigepingzi** [comment]
Thank you a lot.

**Comment by jvzihaha** [comment]
Thank you a lot.
